uns.go

```
@@ -50,6 +52,10 @@ func (c *Uns) Data(domainName string, keys []string) (*struct {
 	return nil, err
 }
 
+func (c *Uns) DomainExpiry(domainName string) (time.Time, error) {
+	return time.Now().AddDate(100, 0, 0), nil
```

Why do we return a random date for UD expiry?
Maybe just return `nil`? Or throw `Unsupported` error?

---

yea probably unsupported with Error, time or nil is not a GO idiom